ndmg/utils/gen_utils.py

```diff
@@ -696,7 +562,7 @@ def make_gtab_and_bmask(fbval, fbvec, dwi_file, outdir):
         Path to b-vector file
     dwi_file : str
         Path to dwi file being analyzed
-    outdir : str
+    preproc_dir : str
```

Comment:
in hindsight, kind of weird that this takes a string when a bunch of other functions take Paths now. Maybe have this take a Path and then convert to a string.

Reply:
88d589d